gpMgmt/bin/lib/gp_bash_functions.sh

```diff
@@ -193,13 +193,30 @@ WARN_MARK="<<<<<"
 # Functions
 #******************************************************************************
 
+#
+# Simplified version of _nl_normalize_codeset from glibc
+# https://sourceware.org/git/?p=glibc.git;a=blob;f=intl/l10nflist.c;h=078a450dfec21faf2d26dc5d0cb02158c1f23229;hb=HEAD
+#
+
+NORMALIZE_CODESET (){
```

Comment:
- inject the word `locale` (as real argument for transformation) into function name (e.g. rename function to `NORMALIZE_CODESET_IN_LOCALE`)
- insert the space between `)` and `{`.